src/terminal/terminal/terminal.h

```diff
@@ -279,6 +289,33 @@ guac_terminal_options* guac_terminal_options_create(
  */
 void guac_terminal_free(guac_terminal* term);
 
+/**
+ * Calculate the available space within the terminal
```

Comment:
Sorry to be such a stickler about style, but block comments should end in periods. 

Also, a little bit more detail about how the function works is useful to understand the function at a glance. In this case, it would be good to mention something about the calculation results being stuffed into the provided int* pointers.